src/pages/Error404Page.vue

```diff
@@ -1,41 +1,208 @@
 <script>
-export default {
+import { defineComponent } from 'vue';
+
+export default defineComponent({
     name: 'Error404Page',
-};
+    computed: {
+        isMobile() {
+            return this.$q.screen.lt.md;
+        },
+        isLandscape() {
+            return this.$q.screen.width > this.$q.screen.height;
+        },
+    },
+    watch: {
+        $route: {
+            immediate: true,
+            handler() {
+                console.log(this.$q.screen);
+                console.log(this.$q.screen.lt);
+                console.log(this.$q.screen.lt.md);
+            },
+        },
+    },
+});
 </script>
 
 <template>
 <div
-    class="fullscreen bg-blue text-white text-center q-pa-md flex flex-center"
+    :class="{
+        'c-404-page': true,
+        'landscape' : isMobile && isLandscape
+    }"
 >
-    <div>
-        <div class="main-text">
-            404
+    <div class="c-404-page__video-wrapper">
+        <video
+            id="bgvid"
+            playsinline
+            autoplay
+            muted
+            loop
+            poster="~/assets/background/Video-top_compressed-poster-00001.jpg"
+            class="c-404-page__video"
+        >
+            <source
+                src="~/assets/background/Video-top_compressed-transcode.webm"
+                type="video/webm"
+            >
+            <source
+                src="~/assets/background/Video-top_compressed-transcode.mp4"
+                type="video/mp4"
+            >
+        </video>
+    </div>
+    <div class="c-404-page__video-overlay" ></div>
+    <div class="c-404-page__video-overlay 404-page__video-overlay--shaded" ></div>
```

Comment:
i think you can remove this div? the --shaded class has no styles anyway

Reply:
I tried but we can't. It seems to be creating some sort of visual tint effect over the video and it is applied twice. If I remove the second div the color of the video changes completely and IDK how to create this double-applied effect using just one div with a modified gradient.

But I guess we can remove the  "404-page__video-overlay--shaded" class.

Note: this code was taken from the Native Layout Page which is very old.